taskvine/src/manager/vine_file.c

```diff
@@ -128,68 +128,72 @@ struct vine_file * retrieve_url(const struct vine_file *f){
 	return 0;
 }
 
-const char * make_mt_cached_name(const struct vine_file *f){
+const char *make_mt_cached_name(const struct vine_file *f){
 
 	unsigned char digest[MD5_DIGEST_LENGTH];
-	char * buffer = vine_task_to_json(f->mini_task);
+	char *buffer = vine_task_to_json(f->mini_task);
 	md5_buffer(buffer,strlen(buffer),digest);
        	return md5_string(digest);
 
 }
-const char * make_url_cached_name(const struct vine_file *f)
+const char *make_url_cached_name(const struct vine_file *f)
 {
 	// DIRECTORY cache names might need the name of the files
-	int val = FOUND_NONE;
+	int val = VINE_FOUND_NONE;
 	int STR_MAX = 256;
 	char line[STR_MAX];
 	char hash_src[STR_MAX];
-	char * buffer;
+	char *buffer;
 	unsigned char digest[MD5_DIGEST_LENGTH];
 
-	char * command = string_format("curl -I -sSL i --verbose --stderr /dev/stdout \"%s\"", f->source);
-	FILE * stream = popen(command, "r");
+	char *command = string_format("curl -I -sSL i --verbose --stderr /dev/stdout \"%s\"", f->source);
+	FILE *stream = popen(command, "r");
 	if(stream == 0) return 0; // curl failed to execute 
 	while(fgets(line, STR_MAX, stream) != NULL){
 		if(sscanf(line, "Content-MD5: %s", hash_src)){
-			val = FOUND_MD5;
+			val = VINE_FOUND_MD5;
 			break;
 		}
 		if(sscanf(line, "content-md5: %s", hash_src)){
-			val = FOUND_MD5;
+			val = VINE_FOUND_MD5;
 			break;
 		}
-		if(val < FOUND_ET && sscanf(line, "ETag: %s", hash_src)){
-			val = FOUND_ET;
+		if(val < VINE_FOUND_ET && sscanf(line, "ETag: %s", hash_src)){
+			val = VINE_FOUND_ET;
 		}
-		if(val < FOUND_ET && sscanf(line, "etag: %s", hash_src)){
-			val = FOUND_ET;
+		if(val < VINE_FOUND_ET && sscanf(line, "etag: %s", hash_src)){
+			val = VINE_FOUND_ET;
 		}
-		if(val < FOUND_LM && sscanf(line, "Last-Modified: %s", hash_src)){
-			val = FOUND_LM;
+		if(val < VINE_FOUND_LM && sscanf(line, "Last-Modified: %s", hash_src)){
+			val = VINE_FOUND_LM;
 		}
-		if(val < FOUND_LM && sscanf(line, "last-modified: %s", hash_src)){
-			val = FOUND_LM;
+		if(val < VINE_FOUND_LM && sscanf(line, "last-modified: %s", hash_src)){
+			val = VINE_FOUND_LM;
 		}
 	}
 	int exit_status = pclose(stream);
-	char * prefix;
+	char *prefix;
 	if(exit_status != 0) return 0; // Curl executes but retuns an error
 	switch(val){
-		case FOUND_NONE:
+		case VINE_FOUND_NONE:
 			return 0;
-		case FOUND_LM:
+			break;
+		case VINE_FOUND_LM:
 			prefix = string_format("LM-md5-");
 			buffer = string_combine(hash_src, f->source); // + server
 			md5_buffer(buffer,strlen(buffer),digest);
                         return string_combine(prefix, md5_string(digest));
```

Comment:
break;

Reply:
Did I misplace the`break`?

Reply:
It is good practice to add a break to each case. Thus:
```C
			return 0;
			break;
		case FOUND_LM:
			break;
		case VINE_FOUND_LM:
			prefix = string_format("LM-md5-");
			buffer = string_combine(hash_src, f->source); // + server
			md5_buffer(buffer,strlen(buffer),digest);
                        return string_combine(prefix, md5_string(digest));
                        break;
```
etc.

One reason is that a future change may remove one of the `return` statements, and the ensuing fall-through will cause a bug difficult to detect.

Reply:
I believe I've added them for each of the four cases.

Reply:
Yes, I see that now. I think I got confused and saw an added `break` to a case that was deleted.